### ISSUE TITLE
Weekly template - how to include dates start/end dates?

### ISSUE THREAD

**Issue by maxmahlke** [jakobkhansen/journal.nvim#2]
Hi, I'd like to edit the template of the `weekly` entry to include the start and end date of the week in question, e.g. `Week 20 - Monday 13/05 -> Sunday 20/05`. I saw that the standard `%B` placeholder sets the correct month for the corresponding week, but adding a `%d` for example gave me today's date (`15`), so I'm not sure how I could get for example the date of the Monday of a given week. Is this possible?

**Comment by jakobkhansen** [comment]
Hi!

In your config for `week` did you remove `date_modifier = "monday"`? This is the default date modifier applied to the entry, otherwise the underlying date will be todays date, which is probably why you are getting `15` instead of `13`.

As for your other question of how to get the range, this is possible with a little more effort. Pull the latest commit (I forgot to pass `date` to template function), and add this config for `week`:

    week = {
        format = '%Y/%m-%B/weekly/week-%W',
        template = function(date)
            local sunday = date:relative({ day = 6 })
            local end_date = os.date('%A %d/%m', os.time(sunday.date))
            return "# Week %W - %A %d/%m -> " .. end_date .. "\n"
        end,
        frequency = { day = 7 },
        date_modifier = "monday"
    },

Basically it gets the next sunday and uses it to produce the `end_date` which contains the string you wanted at the end. This produces the following template for me:

`# Week 20 - Monday 13/05 -> Sunday 19/05`

Does that solve your issue? :)

**Comment by maxmahlke** [comment]
Hi @jakobkhansen, sorry for the delayed response! I finally got time to look into this, and your proposed solution works perfectly. Thanks!
